### PR TITLE
Fix e2e tests

### PR DESCRIPTION
**Pull request by michalina-graczyk**

### CODE REVIEW

**Review by karola312 — APPROVED** [review]

**Comment by karola312** [comment]
https://cloud.cypress.io/projects/o9u948/runs/174/test-results/84a9cfbe-9a01-4e14-962b-5c52e8be04d2 - this test is flaky

**Review by karola312 — CHANGES_REQUESTED** [review]
This test is flaky - https://cloud.cypress.io/projects/o9u948/runs/174/test-results/84a9cfbe-9a01-4e14-962b-5c52e8be04d2

**Review by szczecha — DISMISSED** [review]

File: cypress/e2e/buyProductsAsExistingCustomer.cy.ts [hunk 1]
```diff
@@ -17,11 +25,8 @@ describe("Buy product as existing user", () => {
     cy.addAliasToGraphRequest("user")
       .addAliasToGraphRequest("checkoutShippingAddressUpdate")
       .addAliasToGraphRequest("checkoutBillingAddressUpdate")
-      .addAliasToGraphRequest("checkoutDeliveryMethodUpdate")
-      .get(SHARED_ELEMENTS.productsList)
-      .children()
-      .first()
-      .click();
+      .addAliasToGraphRequest("checkoutDeliveryMethodUpdate");
```
Comment (wojteknowacki) [inline cypress/e2e/buyProductsAsExistingCustomer.cy.ts:28]: Try to keep adding aliases on the very begining of every 'it'. If not alias could not be created fast enough and wait will fail

**Review by wojteknowacki — DISMISSED** [review]

**Review by karola312 — APPROVED** [review]

File: cypress/support/pages/checkout-page.ts [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
 
 export function payByDummyPayment() {
-  cy.addAliasToGraphRequest("order")
-    .get(CHECKOUT_ELEMENTS.paymentMethods)
-    .children()
-    .find("[value='dummy']")
-    .click()
-    .wait(2000);
+  cy.addAliasToGraphRequest("order");
+  cy.get(CHECKOUT_ELEMENTS.paymentProviders).children().find("[value='dummy']").click().wait(2000);
```
Comment (wojteknowacki) [inline cypress/support/pages/checkout-page.ts:7]:
```suggestion
  cy.get(CHECKOUT_ELEMENTS.paymentProviders).find("[value='dummy']").click().wait(2000);
```
is 'children' needed?

**Review by wojteknowacki — APPROVED** [review]

File: cypress/support/pages/checkout-page.ts (hunk 2 above)
Comment (szczecha) [reply cypress/support/pages/checkout-page.ts:7]: Hard to say, It should work without children() but I don't have the environment and capacity to check this now. As the main goal was to fix those tests for the staging demo where we have only Adyen.

**Pull request merged**